packages/quantic/force-app/main/default/lwc/quanticNumericFacet/quanticNumericFacet.js

```diff
@@ -262,9 +264,10 @@ export default class QuanticNumericFacet extends LightningElement {
   }
 
   /**
-   * @param {SearchEngine} engine
+   * @param {SearchEngine | InsightEngine} engine
    */
   initialize = (engine) => {
+    this.engine = engine;
```

Comment:
Why do you prefer this?

Reply:
to avoid passing engine each time as a parameter and more importantly to avoid having some function call `getHeadlessBindings` just to get the engine again while it can be very easily avoided with `this.engine = engine;`